ovirt-engine-keycloak.spec.in

```diff
@@ -32,7 +35,9 @@ Source:		%{name}-@RPM_VERSION@.tar.gz
 
 BuildRequires:	unzip
 
-Requires:	%{name}-setup >= %{version}
+Requires:	%{name}-setup
```

Comment:
Why not >=? You don't want to allow upgrading with an old setup package. But see also bz 2024157.

Reply:
I think I misunderstood how the version locking works, restoring this condition.